specification/iotcentral/resource-manager/Microsoft.IoTCentral/stable/2018-09-01/iotcentral.json

```diff
@@ -733,6 +814,60 @@
           "readOnly": true
         }
       }
+    },
+    "AppTemplate": {
+      "description": "IoT Central Application Template.",
+      "type": "object",
+      "properties": {
+        "manifestId": {
+          "description": "The ID of the template.",
+          "type": "string",
+          "readOnly": true
+        },
+        "manifestVersion": {
+          "description": "The version of the template.",
+          "type": "string",
+          "readOnly": true
+        },
+        "name": {
```

Comment:
>name [](start = 9, length = 4)

`name` is generally considered a reserved property name and intended to be a top level property (see the comment with the json example). You should call it `appTemplateName` instead to avoid user confusion. Microsoft.Authorization RP follows a similar pattern.